lib/manager/tool-versions/extract.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { Fixtures } from '../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+describe('manager/tool-versions/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('', '.tool-versions', {})).toBeNull();
+    });
+
+    it('extracts dep versions for supported tools', () => {
+      const res = extractPackageFile(
+        Fixtures.get('.tool-versions'),
+        'unused_file_name',
+        {}
+      );
+      expect(res).toEqual({
+        deps: [
+          {
+            currentValue: '10.15.0',
+            currentVersion: '10.15.0',
+            datasource: 'github-tags',
+            depName: 'nodejs',
+            lookupName: 'nodejs/node',
+          },
+          {
+            currentValue: '3.2.5',
+            currentVersion: '3.2.5',
+            datasource: 'ruby-version',
```

Comment:
We need to check how they build/compile this version. If they build it directly from repo source, then it should also be github-tags

Reply:
@rarkins as their docs state, they build the version from source by compiling. So I will use `github-tags` in that case as you suggest.